packages/lib/universal/get-feature-flag.ts

```diff
@@ -20,6 +18,10 @@ export const getFlag = async (
 ): Promise<TFeatureFlagValue> => {
   const requestHeaders = options?.requestHeaders ?? {};
 
+  if (!LOCAL_FEATURE_FLAGS[flag]) {
+    return LOCAL_FEATURE_FLAGS[flag];
+  }
+
```

Comment:
🤔

Reply:
I'll remove this.  The billing feature is not disabled when `NEXT_PUBLIC_FEATURE_BILLING_ENABLED` is set as false and you have environmental variables for PostHog available. This was a work around to remove documents limits locally for me to avoid dealing with stripe